src/main/webapp/app/entities/participation-submission/participation-submission.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ParticipationService } from 'app/entities/participation';
+import { ActivatedRoute } from '@angular/router';
+import { Submission } from 'app/entities/submission';
+import { SubmissionService } from 'app/entities/submission/submission.service';
+import { Result } from 'app/entities/result';
+
+import { JhiEventManager } from 'ng-jhipster';
+import { Subscription } from 'rxjs/Subscription';
+
+@Component({
+    selector: 'jhi-participation-submission',
+    templateUrl: './participation-submission.component.html',
+})
+export class ParticipationSubmissionComponent implements OnInit {
+    @Input() participationId: number;
+    submissions: Submission[];
+    eventSubscriber: Subscription;
+    result: Result;
+
+    constructor(
+        private route: ActivatedRoute,
+        private participationService: ParticipationService,
+        private submissionService: SubmissionService,
+        private eventManager: JhiEventManager,
+    ) {}
+
+    ngOnInit() {
+        this.setupPage();
+        this.eventSubscriber = this.eventManager.subscribe('submissionsModification', () => this.setupPage());
+    }
+
+    setupPage() {
+        this.route.params.subscribe(params => {
+            this.participationId = +params['participationId'];
+        });
+
+        this.submissionService.findAllSubmissionsOfParticipation(this.participationId).subscribe(response => {
```

Comment:
There are two things that could be improved here:
- Loading indicator to show while the page is loading (this will be a thing during the lecture with 1000+ entries)
- Error handling (what happens if the request fails?)

E.g. something like this:
```
this.isLoading = true;
this.submissionService.findAllSubmissionsOfParticipation(this.participationId)
.pipe(
  map(({body}) => body),
  catchError(() => of([])) // if the call failed, we return an empty array.
).subscribe(submissions => {
  this.submissions = submissions;
  this.isLoading = false;
});
```